Automattic/jetpack
# RNA Pricing card component: Fix case where prices before and after match

RNA Pricing card component: Fix case where prices before and after match

#### Changes proposed in this Pull Request:

* Updates the `PricingCard` RNA component to display the price only once when price before matches price after.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
* Go to the Storybook folder `cd projects/js-packages/storybook`
* Run `pnpm install && pnpm run storybook:dev`
* Storybook should open in your browser.
* Go to Playground → Pricing Card and set the prices (`priceBefore` and `priceAfter`) to match.
* Verify that only one price gets displayed.

#### Screenshot
<img width="1181" alt="Screenshot 2021-11-16 at 2 43 20 PM" src="https://user-images.githubusercontent.com/1758399/141987389-477738d5-8197-4fc2-a8f6-d0bff96ddfcd.png">
